apis/openstack-nova/src/test/java/org/jclouds/openstack/nova/v2_0/compute/extensions/NeutronSecurityGroupExtensionLiveTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.openstack.nova.v2_0.compute.extensions;
+
+import java.util.Properties;
+import javax.annotation.Resource;
+import javax.inject.Named;
+
+import org.jclouds.Context;
+import org.jclouds.ContextBuilder;
+import org.jclouds.compute.extensions.SecurityGroupExtension;
+import org.jclouds.compute.extensions.internal.BaseSecurityGroupExtensionLiveTest;
+import org.jclouds.compute.reference.ComputeServiceConstants;
+import org.jclouds.config.ContextLinking;
+import org.jclouds.encryption.bouncycastle.config.BouncyCastleCryptoModule;
+import org.jclouds.logging.Logger;
+import org.jclouds.logging.slf4j.config.SLF4JLoggingModule;
+import org.jclouds.openstack.neutron.v2.NeutronApi;
+import org.jclouds.rest.ApiContext;
+import org.jclouds.sshj.config.SshjSshClientModule;
+import org.testng.annotations.Test;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.reflect.TypeToken;
+import com.google.inject.Module;
+
+/**
+ * Live test for openstack-neutron {@link SecurityGroupExtension} implementation.
+ */
+@Test(groups = "live", singleThreaded = true, testName = "NeutronSecurityGroupExtensionLiveTest")
+public class NeutronSecurityGroupExtensionLiveTest extends BaseSecurityGroupExtensionLiveTest {
+
+   @Resource
+   @Named(ComputeServiceConstants.COMPUTE_LOGGER)
+   protected Logger logger = Logger.CONSOLE;
+
+   private Context neutronApiContext;
+
+   public NeutronSecurityGroupExtensionLiveTest() {
+      provider = "openstack-nova";
+
+      Properties overrides = setupProperties();
+      neutronApiContext = ContextBuilder.newBuilder("openstack-neutron")
+               .endpoint(setIfTestSystemPropertyPresent(overrides,
+               "openstack-neutron.endpoint"))
+               .credentials(setIfTestSystemPropertyPresent(overrides,
+               "openstack-neutron.identity"),
+               setIfTestSystemPropertyPresent(overrides, "openstack-neutron.credential"))
```

Comment:
We should be able to use the same Nova properties. It is unlikely to use a neutron that is behind a different keystone instance. Let's use the same endpoint and credentials that are configured for Nova.